packages/components/src/common/postMixin.js

```diff
@@ -69,7 +73,7 @@ export default {
     },
 
     onLinkClick() {
-      this.$emit('click', this.post);
+      this.$emit('click');
```

Comment:
This line is the reason for the failed builds.
Make sure to accept my suggestion to fix the issue.

```suggestion
      this.$emit('click', this.post);
```

Reply:
Ohhh, so this is why. Thank you!